### DISCUSSION TITLE
Do not generate link in inheritance chain if class is excluded

### DISCUSSION THREAD

**Discussion by SmartFoxServer** [dotnet/docfx#10287] [Q&A]
In our project we have the following inheritance chain for the AdvancedManager class:

<img width="413" alt="Screenshot 2024-10-17 alle 17 10 43" src="https://github.com/user-attachments/assets/ee5f7219-ff5e-49ed-be27-348421d947c3">

The BaseManager class is not useful because it only contains internal methods, so we removed it from documentation using the `<exclude/>` tag. Unfortunately DocFx still generates the link in the inheritance chain, which points to a blank page.
Is there a way to avoid this behavior and just show the class name without the link?

**Comment by filzrev** [comment]
The current docfx version does not seem to change reported behavior.

I've added PR #10300 to fix reported issue.
So reported issue will be fixed on next version release.
